tests/integration/functional/artifacts/test_utils.py

```diff
@@ -215,7 +215,11 @@ def artifact_multi_output_metadata_logging_step() -> (
         "description": "Blupus is great!",
         "metrics": {"accuracy": 0.9},
     }
-    log_metadata(metadata=output_metadata, artifact_name="int_output")
+    log_metadata(
+        metadata=output_metadata,
+        artifact_name="int_output",
+        infer_artifact=True,
```

Comment:
Why `infer_artifact=True`? I might be misreading this, but you gave artifact name already, no?

Reply:
I also discussed with @schustmi. Eventually, we do **not** want to enable people to provide just a name to `log_metadata` for the latest version (with models or artifacts). So, if you provide just a name, it needs to be paired with `infer_artifact=True`, so we look for it in the output space.